test/e2e/util.go

```diff
@@ -40,6 +40,7 @@ import (
 	"k8s.io/kubernetes/pkg/apimachinery/registered"
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/cache"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/release_1_1"
```

Comment:
I suggest importing this as "client" (or "vclient", since client is in use)? Should cause less churn when we update to subsequent clients.